projects/plugins/jetpack/modules/masterbar/admin-menu/menu-mappings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Helper mapping between WP Admin pages and WordPress.com
+ *
+ * @package automattic/jetpack
+ */
+
+return array(
+	'dashboard.php'                                  => 'https://wordpress.com/home/',
+	'upload.php?post_type=attachment'                => 'https://wordpress.com/media/',
+	'edit.php?post_type=post'                        => 'https://wordpress.com/posts/',
+	'edit-comments.php'                              => 'https://wordpress.com/comments/',
+	'export.php'                                     => 'https://wordpress.com/export/',
```

Comment:
Is there any way we can make this dynamic? WP Admin is enforced on Export for Atomic sites.

Reply:
I see that visiting `https://wordpress.com/export/` for an Atomic site, suggests visiting WP Admin 
<img width="624" alt="SS 2021-06-25 at 11 18 21" src="https://user-images.githubusercontent.com/12430020/123393741-0f6d2d00-d5a7-11eb-9b5f-0e7ec5982b84.png">

I understand that when you say `dynamic` you mean "Hide the switcher for certain cases", like it doesn't make sense for the Export in Atomic sites to have a switcher to Calypso.

I am not sure whether the complexity that will be introduced is justified by the value of not showing the switcher. 

Do you foresee any problems?

Reply:
>Is there any way we can make this dynamic? WP Admin is enforced on Export for Atomic sites.

@mmtr can you specify what you mean by dynamic here please just so I understand?

If Export is forced in Atomic sites and we want it to behave like Posts (e.g. have the exception) let's consider if the effort and complexity introduced is worth it in terms of time and value. I understand that it is for Posts/Pages due to the extendability for Plugins.

Reply:
I pushed a change, hiding the export for Atomic sites.